torchx/test/torchx/nx_doctest_test.exs

```diff
@@ -80,6 +75,57 @@ defmodule Torchx.NxDoctestTest do
       Torchx.Backend.__unimplemented__()
       |> Enum.map(fn {fun, arity} -> {fun, arity - 1} end)
       |> Kernel.++(@temporarily_broken_doctests)
+      |> Kernel.++(@rounding_error_doctests)
       |> Kernel.++(@inherently_unsupported_doctests)
       |> Kernel.++([:moduledoc])
+
+  describe "rounding error tests" do
```

Comment:
Let's move those to backend_test.exs instead? And keep only doctests here, since they are already a lot. We may also need to break `backend_test.exs` itself into multiple modules (as we discussed) but it can be done another day.